src/lib/transition_router/initial_validator.ml

```diff
@@ -329,6 +350,11 @@ let run ~logger ~trust_system ~verifier ~transition_reader
                   |> Protocol_state.hashes )
                     .state_hash
                 in
+                Internal_tracing.with_state_hash state_hash
+                @@ fun () ->
+                [%log internal] "Failure"
```

Comment:
Likewise here

Reply:
Same as above, it is attached in `~metadata`